### PR TITLE
A/stringify

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: core/arena/stringify.js [hunk 1]
```diff
@@ -12,10 +9,20 @@ import { mapToObject } from './mapToObject'
...
 export const stringify = (object) => {
   if (isNull(object)) return null
-  if (isEmpty(object)) return ''
   return JSON.stringify(object, (key, value) => {
-    if (value.constructor === Map) return mapToObject(value)
-    if (value.constructor === Set) return [...value]
+    if (value.constructor === Map)
+      return {
+        __type: 'Map',
+        __values: JSON.stringify(Array.from(value.entries())),
```
Comment (minotogna) [inline core/arena/stringify.js:16]: use stringify not JSON.stringify

File: core/arena/stringify.js [hunk 2]
```diff
@@ -12,10 +9,20 @@ import { mapToObject } from './mapToObject'
...
+      }
+    if (value.constructor === Set)
+      return {
+        __type: 'Set',
+        __values: JSON.stringify([...value]),
```
Comment (minotogna) [inline core/arena/stringify.js:21]: use stringify not JSON.stringify

File: test/1_unit/005A.stringify.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ const tests = [
   { value: null, valueText: 'null', expected: null },
   { value: undefined, valueText: 'undefined', expected: null },
```
Comment (minotogna) [inline test/1_unit/005A.stringify.js:8]: add at least one test with primitive Number and String

Comment (ramirobg94) [reply test/1_unit/005A.stringify.js:8]: they are on the line 100

**Pull request merged**